lib/vsc/filesystem/gpfs.py

```diff
@@ -65,6 +65,7 @@ def split_output_lines(out):
     Takes into acount that some lines may end in a :, some might not. If the header exhibits no colon,
     the field count might be off and we might not be able to parse the output as expected.
     """
+    out = out.strip().split('\n')
     header_ends_in_colon = out[0][-1] == ":"
```

Comment:
I'm a bit puzzled by this. It can never have worked as far as I can see... `out` always was a string.

Reply:
indeed. but fixing it breaks the code i think. most results from `-Y` will follow the else block now, instead of the if block.
not sure unittests cover this.

Reply:
It now adds a empty entry to the list per line. I'm not sure who this was supposed to work...

Maybe we should just drop it? This change is from 2015 and no one ever noticed that it can never have worked? So is it really needed? In what case does this happen?

Reply:
this part is really odd. why is the percentdecode needed here? should have used the csv parser for this, but now i'mnot sure how or why things are working here ;)

Reply:
There were tests for this but they were not working because of missing `,`.

One is still broken and I'm not sure what the 'correct' result is. If the header ends in a `:`, the the lines below don't: what is supposed to happen? Ignore that empty field completely or add a empty field to all lines?

@itkovian any clue?

Reply:
I must have missed this. Erm, I thought that the out was a list, but it seems it's a string. At least, I guess it is. Bloody duck typing.

Reply:
See #72.